tasks/patches/add-sbom-and-push.yaml

```diff
@@ -43,7 +43,7 @@
   path: /spec/steps/-
   value:
     name: analyse-dependencies-java-sbom
-    image: quay.io/redhat-appstudio/hacbs-jvm-dependency-analyser:70a63556e607e02fe6a19a3f13bb35cedc1f3043
+    image: quay.io/redhat-appstudio/hacbs-jvm-build-request-processor:70a63556e607e02fe6a19a3f13bb35cedc1f3043
```

Comment:
I went back and forth @stuartwdouglas and based on what I saw you doing with the tekton bundle overrides in https://github.com/redhat-appstudio/jvm-build-service/pull/266/files#diff-4003d7cdad9cc5d348e02cd69efa7b36f0565a9f04f1b1615429daa9f7c0b154 I decided the thing to do replace the image vs. removing the step 

of course let me know if I am misinterpreting

and once that PR merges I'll update the commit

Reply:
also removing altogether might have a better chance of affecting the other steps @Michkov updated back in June with #149